docker-hub/builds/index.md

```diff
@@ -212,6 +214,15 @@ If a build fails, a **Retry** icon appears next to the build report line on the
 > If you are viewing the build details for a repository that belongs to an
 > Organization, the Cancel and Retry buttons only appear if you have `Read & Write` access to the repository.
 
+Automated builds have a 4-hour execution time limit. If a build reaches this time limit, it's
+automatically cancelled, and the build logs display the following message:
+
+```text
+2022-11-02T17:42:27Z Build canceled.
```

Comment:
We should probably update this with the new message that @mcapell is adding at the minute 😄